matcher-musings/README.md

```diff
@@ -32,7 +32,7 @@ claim-id = text / int
 
 ```cddl
 RV = { 
-  ? desc: text
+  ? environment-id: any
```

Comment:
I think there are some tradeoffs. To me `any` could improve the flexibility. However, usually `environment-id` is used as a key to index the RV so text might be better.

Reply:
Sure, I just set it to `any` out of pure laziness.  This needs some refinement, with `text` as an obvious candidate.